templates/base.html

```diff
@@ -91,8 +91,15 @@
     >
       <div class="container">
         <a class="navbar-brand" href="index.html">
-          <img class="logo" src="{{config.logo}}" width="180px" />
+          <img
+             class="logo" src="{{config.logo.image}}"
+             height={{config.logo.height | default("auto") | tojson}}
+             width={{config.logo.width | default("auto") | tojson}}
+          />
```

Comment:
use default magic and also added "tojson" to make sure the string is surrounded by double quotes.
The generated html file looks like
```html
        <a class="navbar-brand" href="index.html">
          <img
             class="logo" src="static/images/MiniConf.png"
             height="auto"
             width="180px"
          />
        </a>

```